homeassistant/components/bsblan/climate.py

```diff
@@ -94,14 +94,13 @@ def __init__(
         """Initialize BSBLan climate device."""
         self._current_temperature: Optional[float] = None
         self._available = True
-        self._current_hvac_mode: Optional[int] = None
+        self._hvac_mode: Optional[int] = None
```

Comment:
The home assistant hvac mode is a string, not an integer.

Reply:
Made a mistake that this on not the value from bsblan-library, but the one from home-assistant. Thanks for catching it.